frontend/src/e2e-test/specs/0_citizen/citizen-holiday-reservations.spec.ts

```diff
@@ -197,12 +272,30 @@ describe('Holiday periods', () => {
         })
         .save()
 
+      const child2 = await setupAnotherChild(
+        // Not eligible because placement starts after 2022-01-01
+        LocalDate.of(2022, 3, 1),
```

Comment:
`2022-01-01` could be extracted to a well-named constant and: `foo.addMonths(2)`

Reply:
Fixed and add a few more minor clarifications